test/cppsuite/tests/example_test.cxx

```diff
@@ -38,17 +38,17 @@ class example_test : public test_harness::test {
 
     virtual void
```

Comment:
Do we need the virtual keyword now we have override ?

Reply:
We do not, it seems to work without it, should we make them final too? @tetsuo-cpp

Reply:
We should use final if we dont want any class derived from example_test.cxx to override populate